app/javascript/multi-stamp-tubes/components/MultiStampTubes.vue

```diff
@@ -204,13 +212,14 @@ export default {
     //   ...
     // }
     targetWells() {
-      const wells = {}
-      for (let i = 0; i < this.validTransfers.length; i++) {
-        wells[this.validTransfers[i].targetWell] = {
-          pool_index: this.validTransfers[i].tubeObj.index + 1,
+      return this.validTransfers.reduce((acc, transfer) => {
+        const tubeIndex = transfer.tubeObj.index
+        acc[transfer.targetWell] = {
+          colour_index: this.colourIndex(tubeIndex),
+          human_barcode: this.tubes[tubeIndex].labware.labware_barcode.human_barcode,
         }
-      }
-      return wells
+        return acc
+      }, {})
```

Comment:
Looks like you've changed the structure of the return value here, so could you update the comment above the function to reflect that?

Reply:
Thanks for renaming to colour_index, that seems to make a lot more sense.